bundle/src/main/java/com/cognifide/cq/cqsm/core/scripts/ScriptContent.java

```diff
@@ -52,6 +48,8 @@
 
 	public static final String CQSM_VERIFIED = "cqsm:verified";

```

Comment:
you are introducing new node properties but with old name;  I don't think so if you are going good way... ;)

new type/thing -> new name 'APM'  :)

Reply:
Ok, we develop all the changes related to CQSM->APM transfer on selected branch called `cqsm-to-apm-transfer` and will fix that on that branch :)